sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1276,6 +1276,16 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .booleanConf
     .createWithDefault(false)
 
+  val ORC_STRIPE_SIZE_ROWS = conf("spark.rapids.sql.format.orc.write.stripeSizeRows")
```

Comment:
I think a `lazy val orcStriprSizeRows: Option[Integer] = get(ORC_STRIPE_SIZE_ROWS)` is also needed in `class RapidsConf` to be consistent with other configs even if you don't use it.